packages/db/src/schema.ts

```diff
@@ -812,10 +812,11 @@ export const tamboUsers = pgTable(
       withTimezone: true,
     }),
 
-    // Reactivation email tracking
+    /** @deprecated - Reactivation email feature removed, column kept for data preservation */
     reactivationEmailSentAt: timestamp("reactivation_email_sent_at", {
```

Comment:
you can also just rename this field in the TS:

```suggestion
    deprecatedReactivationEmailSentAt: timestamp("reactivation_email_sent_at", {
```